asf_heat_pump_suitability/analysis/exploratory/aggregate_build_year_per_la.py

```diff
@@ -0,0 +1,44 @@
+"""
+Aggregate the build year data up to local authority level.
+"""
+
+import pandas as pd
+
+if __name__ == "__main__":
+
+    lad_to_lsoa = pd.read_csv(
+        "s3://asf-heat-pump-suitability/source_data/2021_vApr2023_ons_lsoa_to_lad_lookup_EW.csv"
+    )
+    lsoa_build_year = pd.read_csv(
+        "s3://asf-heat-pump-suitability/source_data/2015cdrc_dwelling_ages_E_W.csv"
+    )
+
+    lsoa_to_lad_dict = dict(zip(lad_to_lsoa["LSOA21CD"], lad_to_lsoa["LAD23CD"]))
+    lad_name_dict_dict = dict(zip(lad_to_lsoa["LAD23CD"], lad_to_lsoa["LAD23NM"]))
```

Comment:
```suggestion
    lad_name_dict = dict(zip(lad_to_lsoa["LAD23CD"], lad_to_lsoa["LAD23NM"]))
```
reminds me of dik dik 😆

Reply:
whoops!